src/components/AnalysisModal.js

```diff
@@ -156,19 +157,19 @@ export const AnalysisModal = withDisplayName('AnalysisModal')(
           setCurrentTool(tools.Jupyter.label)
           enterNextViewMode(tools.Jupyter.label)
         }
-      }, [img({ src: jupyterLogoLong, style: _.merge(styles.image, { width: '30%' }) })]),
+      }, [img({ src: jupyterLogoLong, alt: 'Create new notebook', style: _.merge(styles.image, { width: '30%' }) })]),
```

Comment:
It is true that the images need alt text for accessibility. However, these controls are not keyboard accessible-- to see that, try selecting one of them using the keyboard only (using Tab to move between components). Please fix https://broadworkbench.atlassian.net/browse/IA-3043 at the same time.

I would recommend that the components are buttons containing the images, which have alt text. You don't need to have labels on the buttons as long as the alt text is clear.

And it would be good to pull the name of the app/tool from the `tools` variable, so that we aren't duplicating app names/tool names here.

Reply:
Please also add alt text to the Cromwell "button" (line 178).

Reply:
Addressed in [ff9583c](https://github.com/DataBiosphere/terra-ui/pull/2865/commits/ff9583c967be11c71edcd96c97a4a0273a22e06e)